gui/src/lib/process/store.ts

```diff
@@ -35,26 +36,29 @@ export const fetchProcesses = (workspace) => {
       case 'success':
         refetchProcesses(workspace);
         return refetchingResponse(req.data);
+      case 'refetching':
+        // I'm not sure this is right.
+        return req;
```

Comment:
@kendru is this the correct thing to do here?

Reply:
seems reasonable to me, but i think this whole remote data / fetching mechanism needs some love